packages/lib/src/components/internal/Address/Address.tsx

```diff
@@ -120,7 +120,11 @@ export default function Address(props: AddressProps) {
 
     useEffect((): void => {
         const optionalFields = specifications.getOptionalFieldsForCountry(data.country);
+        // Country specific / default schema fields, include both required and optional fields.
+        const validSchemaFields = specifications.getAddressSchemaForCountryFlat(data.country);
         const processedData = ADDRESS_SCHEMA.reduce((acc, cur) => {
+            if (!validSchemaFields?.includes(cur)) return acc;
```

Comment:
Don't we need to also add the value of `firstName` to acc?
We just want to avoid line 134?

Reply:
No we don't want to add `firstName` and `lastName` for all the schemas, because that caused the issue [here](https://github.com/Adyen/adyen-web/pull/2591). 
Only if the schema explicitly mentions to show `firstName` & `lastName` for instance [Riverty](https://github.com/Adyen/adyen-web/blob/3ce12ef069ba39c4ae9c5f794261f5bbffb87e9a/packages/lib/src/components/Riverty/config.ts#L7) we put the key and value to the `acc`.

Reply:
Makes sense sorry. In my head I was just trying to avoid the `N/A` default scenario. If you don't pass things that are not in the schema also works.